velox/exec/StreamingWindowBuild.cpp

```diff
@@ -61,10 +163,21 @@ void StreamingWindowBuild::noMoreInput() {
 }
 
 std::unique_ptr<WindowPartition> StreamingWindowBuild::nextPartition() {
+  currentPartition_++;
+
+  if (currentPartition_ < merges_.size() &&
+      merges_[currentPartition_] != nullptr) {
+    loadNextPartitionFromSpill();
+    VELOX_CHECK(!spilledSortedRows_.empty(), "No window partitions available")
+    auto partition =
+        folly::Range(spilledSortedRows_.data(), spilledSortedRows_.size());
+    return std::make_unique<WindowPartition>(
```

Comment:
It seems that the data of a partition is preset here to be fully stored in memory. But in the case of spill, the data that comes back from the spill file often cannot fit into memory. What if part of it is in memory and part of it is in the spill file.

Reply:
@guhaiyan0221 Currently, only one window partition is supported, either it is in memory or it has been spilled. It does not support a part in memory and a part has been spilled. The reason why the data needs to be loaded back into memory here is because the  `WindowPartition` only accepts data in memory and does not accept spilled data. This is also the second step to do. Thanks.